docs/rfcs/XXX-pull-request-review.md

```diff
@@ -0,0 +1,145 @@
+
+## Rationale and alternatives
+
+One alternative may be to show review comments _only_ within the "changes" tab of an `IssueishPaneItem`. This simplifies flow considerably, because it removes the need to provide navigation among different views of the same review, and unifies the handling of current and non-current pull requests. However, I believe that the renderings of reviews in all three places each serve a unique purpose:
+
+* Reviews within the "Changes" tab of the `IssueishPaneItem` reveal a narrative formed by all of the reviews on a specific pull request together, as a conversation among reviewers.
+* Reviews within the "Review" tab of the `IssueishPaneItem` reveal the narrative flow within each review individually. For example, review comments that refer to other comments within the same review (e.g. "same here" or "and again") become clearer here.
+* Review comments within open TextEditors allow the reader to use more context within the source code to evaluate, address, or respond to each individual comment thread: consistency with functions that are not visible within the immediate diff, context within algorithms that span many lines. They also allow the receiver of a review to preserve the mental context of the review communication as they move back and forth between reading the content of a review and applying it to their source.
+
+## Unresolved questions
+
+### Questions I expect to address before this is merged
+
+Can we access "draft" reviews from the GitHub API, to unify them between Atom and GitHub?
+
+How do we represent the resolution of a comment thread? Where can we reveal this progress through each review, and of all required reviews?
+
+Are there any design choices we can make to lessen the emotional weight of a "requests changes" review? Peer review has the most value when it discovers issues for the pull request author to address, but accepting criticism is a vulnerable moment.
+
+Similarly, are there any ways we can encourage empathy within the review authoring process? Can we encourage reviewers to make positive comments or demonstrate humility and open-mindedness?
+
+### Questions I expect to resolve throughout the implementation process
+
+Review comment positioning within live TextEditors will be a tricky problem to address satisfactorily. What are the edge cases we need to handle there?
+
+The GraphQL API paths we need to interact with all involve multiple levels of pagination: pull requests, pull request reviews, review comments. How do we handle these within Relay? Or do we interact directly with GraphQL requests?
+
+How do we handle comment threads?
+
+### Questions I consider out of scope of this RFC
+
+What other pull request information can we add to the GitHub pane item?
+
+Are there other tabs that we need within the `IssueishPaneItem`?
```

Comment:
Personally I think we might already have too many tabs at this point >_<. That said, of course if we find good reason to add another I'm 👍

Reply:
Hah, that was honestly kind of what I was getting at here. If we can think ahead to even _more_ tabs we'd want to add, it would be another reason not to add two here.